### DISCUSSION TITLE
Run namma yatri frontend in local system

### DISCUSSION THREAD

**Discussion by Ranit-Bandyopadhyay** [nammayatri/nammayatri#6487] [Q&A]
Hello,
I have tried doing setup for namma yatri in local system for frontend part, but giving an error:
ERROR in ./index.js
Module not found: Error: Can't resolve './output/ConfigProvider/index.js' in '/home/runner/VerifiableFittingExabyte/nammayatri/Frontend/ui-driver'
 @ ./index.js 7:0-65 60:0-13

ERROR in ./index.js
Module not found: Error: Can't resolve './output/Engineering.Helpers.LogEvent/index.js' in '/home/runner/VerifiableFittingExabyte/nammayatri/Frontend/ui-driver'
 @ ./index.js 8:0-81 375:6-21

ERROR in ./index.js
Module not found: Error: Can't resolve './output/Main' in '/home/runner/VerifiableFittingExabyte/nammayatri/Frontend/ui-driver'
 @ ./index.js 6:0-44 146:4-34 158:4-26 226:8-23 228:8-23 231:10-32 237:10-32 241:10-32 244:8-23 313:4-34 315:4-32 317:4-19 337:6-24 341:6-36 344:4-34 352:4-35 357:6-36

Can anyone help

**Comment by witcher-shailesh** [comment]
Hi @Ranit-Bandyopadhyay, have you done the the nix setup? Also can you help me with the command on which you're facing the mentioned error?

**Reply by Ranit-Bandyopadhyay** [reply]
yes i have done the nix setup. Next when i do npm start then the error is coming

**Reply by kranthik10** [reply]
Hey @Ranit-Bandyopadhyay Can you help me to setup nix on windows as am a unity game developer i dont want to load another Distro.
